pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt

```diff
@@ -0,0 +1,38 @@
+Explaining connectivity from 10.240.40.184 to 10.240.20.141 within vpc0
+Interpreted source: InstanceId:7[10.240.40.184]
+Interpreted destination: InstanceId:15[10.240.20.141]
+=======================================================================
+
+Connections from InstanceId:7[10.240.40.184] to InstanceId:15[10.240.20.141]: All Connections
+
+Path:
+	InstanceId:7[10.240.40.184] -> security group GroupName:11 -> network ACL NetworkAclId:48 -> dashoard -> 
```

Comment:
dashoard -> dashboard ?

Reply:
I added one explain example, w/o debugging it, just for start fixing...